packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -167,4 +167,45 @@ describe('FullStory', () => {
       })
     })
   })
+
+  describe('identifyUserV2', () => {
+    it('makes expected request with default mappings', async () => {
+      nock(baseUrl).post(`/v2beta/users?${integrationSourceQueryParam}`).reply(200)
+      const event = createTestEvent({
+        type: 'identify',
+        userId,
+        anonymousId,
+        traits: {
+          email,
+          name: displayName,
+          'originally-hyphenated': 'some string',
+          'originally spaced': true,
+          'originally.dotted': 1.23,
+          typeSuffixed_bool: true
+        }
+      })
+
+      const [response] = await testDestination.testAction('identifyUserV2', {
+        settings,
+        event,
+        useDefaultMappings: true
+      })
+
+      expect(response.status).toBe(200)
+      expect(JSON.parse(response.options.body as string)).toEqual({
+        uid: userId,
+        email,
+        display_name: displayName,
+        properties: {
+          email,
+          name: displayName,
+          segmentAnonymousId: anonymousId,
+          originallyhyphenated: 'some string',
+          originallyspaced: true,
+          originallydotted: 1.23,
+          typeSuffixed_bool: true
```

Comment:
This seems like the correct behavior (not to strip a type suffix if one is given). To be sure, the v2 API will gracefully handle given type suffixes even though they're no longer required, correct?

Reply:
> Please be sure to not add type suffixes though. _str gets treated as part of the field name in the V2 API.

Based on @patrick-fs comment above type suffixes will be ignored and treated as part of the field name.